tests/moduletool/test_python_dependencies.py

```diff
@@ -54,3 +55,29 @@ def test_collect_python_requirements(tmpdir):
     reqs = project.collect_python_requirements()
     expected_reqs = ["iplib@git+https://github.com/bartv/python3-iplib", "pytest>=1.5", "iplib>=0.0.1", "dummy-yummy"]
     assert sorted(reqs) == sorted(expected_reqs)
+
+
+def test_requirements_from_source_info(tmpdir):
+    common.makeproject(tmpdir, "test-project", deps=[("mod1", "")], imports=["mod1"])
```

Comment:
Add a docstring to indicate that this test case verifies the requirements resolution code path used by the exporter. The code case above this one does the same for the code path used by the compiler. This difference was not immediately clear to me.